### PR TITLE
Fix flags, add subcommand aliases

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

File: cmd/info.go [hunk 1]
```diff
@@ -3,12 +3,10 @@ package cmd
...
 
-	"github.com/Benchkram/errz"
 	"github.com/puppetlabs/wash/api/client"
 	cmdutil "github.com/puppetlabs/wash/cmd/util"
 	"github.com/puppetlabs/wash/config"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 )
 
 func infoCommand() *cobra.Command {
```
Comment (ekinanp) [inline cmd/info.go]: Could we alias this to `winfo` ?

Comment (MikaelSmith) [reply cmd/info.go]: Hmm, in my case it's not builtin, it's just at /usr/bin/info. Which shell are you using?

Comment (ekinanp) [reply cmd/info.go]: Oh, I meant mine is also at `/usr/bin/info`. I'm using Bash.

If `info` isn't used that often, OK to keep it as-is

Comment (MikaelSmith) [reply cmd/info.go]: It seems to be an alternative to `man`. I've never used it.

Comment (ekinanp) [reply cmd/info.go]: May as well alias it then. `winfo` is still pretty short

Comment (MikaelSmith) [reply cmd/info.go]: The same argument can be made for `clear`. Should I change that back to `wclear` as well, since it doesn't do the same thing?

Comment (MikaelSmith) [reply cmd/info.go]: Guess that makes sense. Done.

Comment (ekinanp) [reply cmd/info.go]: Yeah, definitely for `clear`. I use clear a lot in my shell.

File: cmd/rootMain.go [hunk 2]
```diff
@@ -53,6 +53,7 @@ func runShell(cachedir, mountpath string) exitCode {
 		"wexec":    "exec",
 		"find":     "find",
 		"help":     "help",
+		"winfo":    "info",
```
Comment (ekinanp) [inline cmd/rootMain.go]: It'd be nice if this map were generated by the RootCommand so I don't have to remember to update this whenever I add a new subcommand, but it's not a blocker or that big of a deal.

Comment (MikaelSmith) [reply cmd/rootMain.go]: I started doing that, then decided I didn't want to include all of them so left it out.

**Pull request merged**